### PR TITLE
MGMT-18959: Change the latest release image used for e2e jobs tp 4.18

### PR DESCRIPTION
**Pull request by danmanor**

### CODE REVIEW

**Comment by danmanor** [comment]
/pj-rehearse pull-ci-openshift-assisted-installer-master-edge-e2e-metal-assisted pull-ci-openshift-assisted-test-infra-master-e2e-metal-assisted pull-ci-openshift-assisted-image-service-main-edge-e2e-ai-operator-ztp

**Comment by danmanor** [comment]
/pj-rehearse pull-ci-openshift-assisted-installer-master-edge-e2e-metal-assisted pull-ci-openshift-assisted-test-infra-master-e2e-metal-assisted pull-ci-openshift-assisted-image-service-main-edge-e2e-ai-operator-ztp

**Comment by danmanor** [comment]
/pj-rehearse pull-ci-openshift-assisted-image-service-main-edge-e2e-ai-operator-ztp

**Comment by danmanor** [comment]
/pj-rehearse pull-ci-openshift-assisted-image-service-main-edge-e2e-ai-operator-ztp

**Comment by danmanor** [comment]
/pj-rehearse pull-ci-openshift-assisted-installer-master-edge-e2e-metal-assisted pull-ci-openshift-assisted-test-infra-master-e2e-metal-assisted pull-ci-openshift-assisted-image-service-main-edge-e2e-ai-operator-ztp

File: ci-operator/config/openshift/assisted-image-service/openshift-assisted-image-service-main.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ releases:
   latest:
     integration:
       include_built_images: true
-      name: "4.17"
+      name: "4.18"
```
Comment (eifrach) [inline ci-operator/config/openshift/assisted-image-service/openshift-assisted-image-service-main.yaml:10]: can't we call this latest  and set all the jobs to latest

and next time just update the version 
i.e 
```suggestion
      name: "latest"
```
job example
```
- as: e2e-metal-assisted-cnv-latest
  optional: true
  steps:
```

Comment (danmanor) [reply ci-operator/config/openshift/assisted-image-service/openshift-assisted-image-service-main.yaml:10]: I don't think there is an image stream named "latest" -
```
grep -C 3 -r "integration:" ./ | grep "name:" | grep latest
./ci-operator/config/quay/quay-bridge-operator/quay-quay-bridge-operator-master__ocp-latest.yaml-      name: "4.13"
./ci-operator/config/quay/quay-operator/quay-quay-operator-master__ocp-latest.yaml-      name: "4.13"
./ci-operator/config/quay/quay-operator/quay-quay-operator-redhat-3.8__ocp-latest.yaml-      name: "4.13"
./ci-operator/config/quay/quay-operator/quay-quay-operator-redhat-3.9__ocp-latest.yaml-      name: "4.13"
```

Comment (danmanor) [reply ci-operator/config/openshift/assisted-image-service/openshift-assisted-image-service-main.yaml:10]: it must be x.y version

Comment (danmanor) [reply ci-operator/config/openshift/assisted-image-service/openshift-assisted-image-service-main.yaml:10]: https://steps.ci.openshift.org/ci-operator-reference#line1266

**Pull request merged**